### PR TITLE
Eslint and stricter testing

### PR DESCRIPTION
**Pull request by cefn**
Mainly getting to know the repo, linting etc.

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+12.4.0
```
Comment (CarwynNelson) [inline .node-version]: This should probably be node 10, since that is what we are using elsewhere.

File: __tests__/schemas/schema.test.js [hunk 2]
```diff
@@ -1,13 +1,44 @@
-/* eslint-disable node/no-unpublished-require */
 const Ajv = require('ajv');
-const schema = require('../../src/schemas/schema.json')
+const {
+  readFileSync
```
Comment (CarwynNelson) [inline __tests__/schemas/schema.test.js]: is this the formatting that the eslint rules forced?

Comment (cefn) [reply __tests__/schemas/schema.test.js]: Yes we moved away from requires in favour of explicit file loading and parsing, partly led by eslint complaining about it.

File: .node-version (hunk 1 above)
Comment (cefn) [reply .node-version]: Only pinned at this version while it's a tool run locally, allowing us to have contact with latest node. Production deployment would indeed mean repinning.

**Pull request merged**